tests/end_to_end/test_remote_fetch.py

```diff
@@ -42,6 +45,11 @@ def _assertLogsWritten(self):
         # client should have gotten some reads back
         self.assertEqual(len(self.client.getOutLines()), 10)
 
+    def tearDown(self):
+        # clean up downloaded local index file
+        for indexFile in glob.glob("*.bai"):
```

Comment:
they get written in the current directory.  let's put this in as a stop-gap and then file another issue to take care of this for real.